interfaces/acados_matlab_octave/ocp_set_ext_fun.m

```diff
@@ -497,7 +600,7 @@
             setenv('CFLAGS', cflags_tmp);
             mex(acados_include, acados_interfaces_include, external_include, blasfeo_include,...
                 hpipm_include, acados_lib_path, acados_matlab_octave_lib_path, model_lib_path, '-lacados',...
-                 '-lhpipm', '-lblasfeo', ['-l', model_name], mex_files{1});
+                 '-lhpipm', '-lblasfeo', ['-l', model_name], mex_files{1}, extra_sources{ii});
```

Comment:
Hi @mindThomas 

I just tried your changes. It works fine in Matlab.
However, this line is breaking the interface for Octave.
Apparently, it is not possible compile multiple C files at once.

From the Octave documentation:

>  `mex [options] file `
> Compile source code written in C, C++, or Fortran, to a MEX file.
> This is equivalent to `mkoctfile --mex [options] file`.

Octave compatibility is especially important, because we run the tests using Octave on travis.
I think, it would be the cleanest to compile the "manual" function into the model library, i.e. here:
https://github.com/acados/acados/blob/ea30ca76e6daa0e550b97933dadee9b19ff558fe/interfaces/acados_matlab_octave/ocp_generate_casadi_ext_fun.m#L188

Alternatively, one could probably compile it separately and link.
What do you think?